src/test/java/org/xstefank/api/TeamCityAPITest.java

```diff
@@ -0,0 +1,12 @@
+package org.xstefank.api;
+
+import org.junit.Test;
+
+public class TeamCityAPITest {
+
+    @Test(expected = NullPointerException.class)
+    public void testTriggerBuildNullParams() {
+        TeamCityAPI teamCityAPI = new TeamCityAPI(null, 0, null, null, null);
```

Comment:
maybe you can provide testing backing configuration file to avoid the need for this constructor? not necessary though